include/librealsense2/hpp/rs_processing.hpp

```diff
@@ -526,6 +526,35 @@ namespace rs2
         }
     };
 
+    class nv12_decoder : public filter
+    {
+    public:
+        /**
+        * Creates YUY decoder processing block. This block accepts raw YUY frames and outputs frames of other formats.
```

Comment:
Rewrite please (reference to native NV12 and state differences)

Reply:
Done